recipesearch/src/Recipes/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+export class SearchResults extends Component {
+  render() {
+    const { searchResults } = this.props;
+    console.log(searchResults)
```

Comment:
Avoid committing `console.log()`s to origin

Reply:
Thanks for pointing this out, will remove it right away

Reply:
I have remove it